### PR TITLE
Wazuh indexer and dashboard release

### PR DESCRIPTION
**Pull request by vcerenu**
This PR modifies the Wazuh manager stack display, using `wazuh-indexer` and `wazuh-dashboard` instead of opendistro and kibana.
In addition, all the variables are modified to reflect that they are referencing the new Wazuh applications.
Closes https://github.com/wazuh/wazuh-kubernetes/issues/216

### CODE REVIEW

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Review by teddytpc1 — APPROVED** [review]
LGTM

File: wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
 spec:
   selector:
-    app: wazuh-elasticsearch
+    app: wazuh-indexer
   ports:
-    - name: es-nodes
-      port: 9300
-      targetPort: 9300
+    - name: indexer-nodes
+      port: 9800
```
Comment (alberpilot) [inline wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml:22]: The port to be used is the Elasticsearch default one, 9300, right?

File: wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml [hunk 2]
```diff
@@ -5,20 +5,20 @@
...
+      targetPort: 9800
```
Comment (alberpilot) [inline wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml:23]: Same?

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml (hunk 2 above)
Comment (teddytpc1) [reply wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml]: Corrected.

File: wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml (hunk 1 above)
Comment (teddytpc1) [reply wazuh/indexer_stack/wazuh-indexer/indexer-svc.yaml]: Corrected.

**Review by alberpilot — APPROVED** [review]

**Pull request merged**